physics/src/sphere.rs

```diff
@@ -19,9 +21,12 @@ impl SphereColider {
     }
 
     pub fn get_radius(&self, scale: Vec3) -> f32 {
-        let scale = scale.x; // TODO FIX, this is left to the user to discover
+        // TODO: add support for non-uniformly scaled "spheres"
+        let scale = (scale.x + scale.y + scale.z) / 3.;
```

Comment:
nice xD
I'm a bit out of the loop regarding the physics, but have we thought about using capsule colliders for ellipsoids? intuitively I feel like it should be the cheapest option but idk?

Reply:
No, I dont want to add another shape atm